src/integrationTest/resources/zipcontents/fewer_pdfs_than_declared/metadata.json

```diff
@@ -14,7 +14,7 @@
+        "ocr_data": "ewogICJNZXRhZGF0YV9maWxlIjogWwogICAgIHsgIm1ldGFkYXRhX2ZpZWxkX25hbWUiOiAicmlnaHQgdG8gYXBwZWFsX01ybiIsICJtZXRhZGF0YV9maWVsZF92YWx1ZSI6ICJGYWxzZSIgfSwKICAgICB7ICJtZXRhZGF0YV9maWVsZF9uYW1lIjogImNvbnRhaW5zX01ybiIsICJtZXRhZGF0YV9maWVsZF92YWx1ZSI6ICJUcnVlIiB9LAogICAgIHsgIm1ldGFkYXRhX2ZpZWxkX25hbWUiOiAiYmVuZWZpdFR5cGVfZGVzY3JpcHRpb24iLCAibWV0YWRhdGFfZmllbGRfdmFsdWUiOiAiQ2l2aWwgRGl2b3JjZSBDYXNlIn0sCiAgICAgeyAibWV0YWRhdGFfZmllbGRfbmFtZSI6ICJhcHBlbGxhbnRfdGl0bGUiLCAibWV0YWRhdGFfZmllbGRfdmFsdWUiOiAiTXIifSwKICAgICB7ICJtZXRhZGF0YV9maWVsZF9uYW1lIjogImFwcGVsbGFudF9maXJzdE5hbWUiLCAibWV0YWRhdGFfZmllbGRfdmFsdWUiOiAiU3VkaGlyIFPMtmjMtnLMtmXMtnPMtnTMtm7MtmHMtiJ9LAogICAgIHsgIm1ldGFkYXRhX2ZpZWxkX25hbWUiOiAiYXBwZWxsYW50X2xhc3ROYW1lIiwgIm1ldGFkYXRhX2ZpZWxkX3ZhbHVlIjogIlNocmVzdG5hIn0sCiAgICAgeyAibWV0YWRhdGFfZmllbGRfbmFtZSI6ICJhcHBlbGxhbnRfYWRkcmVzcyIsICJtZXRhZGF0YV9maWVsZF92YWx1ZSI6ICIyNCBPbWVnYSBTdC4gdV9rXyBMb25kb24gVUsifSwKICAgICB7ICJtZXRhZGF0YV9maWVsZF9uYW1lIjogImFwcGVsbGFudF9wb3N0Y29kZSIsICJtZXRhZGF0YV9maWVsZF92YWx1ZSI6ICJENDE3QlMifSwKICAgICB7ICJtZXRhZGF0YV9maWVsZF9uYW1lIjogImFwcGVsbGFudF9waG9uZSIsICJtZXRhZGF0YV9maWVsZF92YWx1ZSI6ICIwNzg5NDQ5MTEwNiJ9LAogICAgIHsgIm1ldGFkYXRhX2ZpZWxkX25hbWUiOiAiYXBwZWxsYW50X21vYmlsZSIsICJtZXRhZGF0YV9maWVsZF92YWx1ZSI6ICIifSwKICAgICB7ICJtZXRhZGF0YV9maWVsZF9uYW1lIjogImFwcGVsbGFudF9kb2IiLCAibWV0YWRhdGFfZmllbGRfdmFsdWUiOiAiMTEvMTAvMTk4MSJ9LAogICAgIHsgIm1ldGFkYXRhX2ZpZWxkX25hbWUiOiAiYXBwZWxsYW50X25pRmFsc2UiLCAibWV0YWRhdGFfZmllbGRfdmFsdWUiOiAiNDdCQVM3NjU0In0sCiAgICAgeyAibWV0YWRhdGFfZmllbGRfbmFtZSI6ICJpc19hcHBvaW50ZWUiLCAibWV0YWRhdGFfZmllbGRfdmFsdWUiOiAiVHJ1ZSJ9LAogICAgIHsgIm1ldGFkYXRhX2ZpZWxkX25hbWUiOiAiYXBwb2ludGVlX3RpdGxlIiwgIm1ldGFkYXRhX2ZpZWxkX3ZhbHVlIjogIk1yIn0sCiAgICAgeyAibWV0YWRhdGFfZmllbGRfbmFtZSI6ICJhcHBvaW50ZWVfZmlyc3ROYW1lIiwgIm1ldGFkYXRhX2ZpZWxkX3ZhbHVlIjogIlNvcGhpYSJ9LAogICAgIHsgIm1ldGFkYXRhX2ZpZWxkX25hbWUiOiAiYXBwb2ludGVlX2xhc3ROYW1lIiwgIm1ldGFkYXRhX2ZpZWxkX3ZhbHVlIjogIlNvbGUifSwKICAgICB7ICJtZXRhZGF0YV9maWVsZF9uYW1lIjogImFwcG9pbnRlZV9hZGRyZXNzIiwgIm1ldGFkYXRhX2ZpZWxkX3ZhbHVlIjogIk4vQSJ9LAogICAgIHsgIm1ldGFkYXRhX2ZpZWxkX25hbWUiOiAiYXBwb2ludGVlX3Bvc3Rjb2RlIiwgIm1ldGFkYXRhX2ZpZWxkX3ZhbHVlIjogIkQxNzFZNyJ9LAogICAgIHsgIm1ldGFkYXRhX2ZpZWxkX25hbWUiOiAiYXBwb2ludGVlX2RvYiIsICJtZXRhZGF0YV9maWVsZF92YWx1ZSI6ICIxNC8xMC8xOTgwIn0sCiAgICAgeyAibWV0YWRhdGFfZmllbGRfbmFtZSI6ICJhcHBvaW50ZWVfbmlGYWxzZSIsICJtZXRhZGF0YV9maWVsZF92YWx1ZSI6ICI0NDE3NjU0M0IifQogIF0KfQo=",
```

Comment:
Same data, but in a valid JSON. Previous version has some double quotes missing and an extra comma.